target/linux/qualcommax/ipq807x/base-files/lib/upgrade/mmc.sh

```diff
@@ -1,83 +0,0 @@
-#
-# Copyright (C) 2016 lede-project.org
```

Comment:
This should go into separate commit as well.

Reply:
this change was lumped in this commit because it was the last device using this non-standard code, assuming the previous 2 commits were merged.